### DISCUSSION TITLE
Red links

### DISCUSSION THREAD

**Discussion by Martien** [observablehq/framework#1828] [Ideas]
What is the best way to have Framework support “red links”, like in Wikipedia: A **[red link](https://en.wikipedia.org/w/index.php?title=Red_link_example)**, signifies that the linked-to page does not exist—it either never existed, or previously existed but has been deleted.

I know `markdown-it` is available. Is that the best route?

**Comment by Fil** [comment]
Also discussed in https://github.com/observablehq/framework/discussions/1858#discussioncomment-11437041 and #1730.
